### PR TITLE
Deprecate old serializer mixins

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**

### CODE REVIEW

File: relnotes/serializer.deprecation.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Deprecation of contiguous serializer mixins
+
+`ocean.util.serialize.contiguous.model.LoadCopyMixin`
+`ocean.util.serialize.model.VersionDecoratorMixins`
+
+Both module are not used as part of serializer/decorator implementation as of
+ocean v4.0.0 and now are completely deprecated. All symbols defined there (like
+`VersionHandlingException` can also be found in other serializer/decorator
```
Comment (gavin-norman-sociomantic) [inline relnotes/serializer.deprecation.md]: Missing )

File: src/ocean/util/serialize/contiguous/model/LoadCopyMixin.d [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 *******************************************************************************/
 
-module ocean.util.serialize.contiguous.model.LoadCopyMixin;
+deprecated module ocean.util.serialize.contiguous.model.LoadCopyMixin;
```
Comment (gavin-norman-sociomantic) [inline src/ocean/util/serialize/contiguous/model/LoadCopyMixin.d:17]: Message?

Comment (mihails-strasuns-sociomantic) [reply src/ocean/util/serialize/contiguous/model/LoadCopyMixin.d:17]: Not supported for modules in D1

Comment (gavin-norman-sociomantic) [reply src/ocean/util/serialize/contiguous/model/LoadCopyMixin.d:17]: Ah, of course.

**Comment by mihails-strasuns-sociomantic** [comment]
Green

**Review by gavin-norman-sociomantic — APPROVED** [review]

**Pull request merged**